### ISSUE TITLE
Studies with issues before 2022

### ISSUE THREAD

**Issue by KaimingTao** [hivdb/covid-drdb-payload#37]
- [x] Acevedo21, Supplementary Figure 1b, we can distinguish the data by age. _vector image_
- [x] Alter21,  Extended Data Fig. 1 right part. _bitmap_
- [x] Arien21, Figure 1B
- [x] Annavajhala21b, Figure 3B. _bitmap_
- [x] Arora21b, Figure 1E, 1F
- [x] Arunachalam21, Figure 1c. _vector image_
- [x] Bates21b, Figure 2, _vector_, not sure if blue and yellow dots paired, please also get the age from x-axis
- [x] Behrens21, figure 1A, 1B. _bitmap_, unpaired
- [x] Collier21b, Figure 1d, 1f, _bitmap_, please ignore _HS_ (outlier data)
- [x] Carreño21, Figure 3b, 3c, 3d
- [x] Cele21b, Figure 1E, 1F, 1G, 2B, 2C, 2D, 2E, 2F, 2G, 2H, 2I
- [x] Choi21, Figure 1B, vector image, paired
- [x] Choi21b, Fig 3a, 3b, 4a, 4b, 4c, Figure S1
- [x] Diez21, Figure 1 50% y-axis value. _serial dilution figure_
- [x] Falsey21,10.1056/NEJMc2113468, Figure 1A, 1B, unpaired, _vector_
- [x] Fabricius21, Figure 2D, Figure 5B, 5C, 5D, unpaired, _vector_
- [x] Fernandez21, Figure 1B, _bitmap_. unpaird
- [x] Geers21, Figure 2F. _vector image_
- [x] Gong21, Fig. 6, _vector_, paired by shape, 5 data points for each bar.
- [x] Gushchin21, Figure 3 2nd row. _bitmap_, all figures have different numbers of paired points.
- [x] Haslwanter21, Fig 4A, _bitmap_, curve figure
- [x] Kitchin21, Fig S2A, S2B
- [x] Ledesma21, Figure 2A, Figure 2B. _bitmap_
- [x] Leier21, Figure 2A, 2B, 2C, 2D, _bitmap_, unpaired
- [x] Li21c, Figure 4B, _bitmap_, unpaired
- [x] Lassauniere21, Figure 2, paired; Figure 3 only Alpha, Beta, Gamma, unpaired
- [x] McCallum21c, Figure 1A, 1B, 1C. _bitmap_, these data can get from Figure 2
- [x] Miyakawa21, Figure 1E, unpaired points, 150 points, _vector image_
- [x] Miyakawa21b, Figure 1A, Supplementary Figure S2, _vector image_
- [x] Mishra21, Figure 2d, 3d
- [x] Moriyama21, Figure 5D, _vector_, paired points, please see the y-axis use func log2(NT/5)
- [x] Mlcochova21, Figure 1c (_bitmap_, unpaired), Figure 1d (_bitmap_, unpaired), Figure 1e
- [x] Muller21, Figure 1, the number in a circle means how many results have the same value, you can use the cumulative_count to record same pairs
- [x] Neumann21, Figure 5, _bitmap_, unpaired data, for VOC (please distinguish filled green dot and empty green dot in rx_name)
- [x] Neumann21, Figure 4a, please differentiate two types of green dots
- [ ] Ngugen21, Figure 1A-F, _bitmap_ (if it's hard to do please contact @KaimingTao). Figure 1G is not possible to extract data (ignore it).
- [x] Neerukonda21,10.1101/2021.11.05.467523, Figure 1B, 1C, Supplementary Figure S1A, Supplementary Figure S1B
- [x] Pannus21, Figure 2e. _vector image_
- [x] Parry21, Figure 4a, _bitmap_
- [x] Payne, Figure 2A, 5A
- [x] Ren21, Figure 3a, 3b, 3c
- [x] Reynolds21, Figure 1G, Figure 2A, Figure 2B. _vector image_
- [x] Riou21, Figure 2C. _vector image_
- [x] Roessler21, Figure 1 a to g
- [x] Sahin20, Figure 2C, _vector image_
- [x] Sahin21, Figure 1b (_vector image_),  Extended Data Fig. 4d (_bitmap_, unpaird)
- [x] Scheid21, Figure 7C, D, E, F, G, H
- [x] Spencer21, Figure 2C, 3C, 5D, _bitmap_, points maybe overlap
- [x] Stadtmuller21, Fig 5B
- [x] Suthar21, Figure 3a, _bitmap_, paired
- [x] Tada21, Figure 2D, _bitmap_, unpaired points
- [x] Trombetta21, Figure 1C, _bitmap_, unpaired points
- [x] Tea21, Figure 5H, 5K, 5L, rasterize imaged, paired points in the middle, individual points from left and right. _bitmap_
- [x] Vidal21, Figure 4B. _bitmap_, Figure 4A, unpaired
- [x] Vacharathit21, Supplementary Figure 1A or 1B, _bitmap_, unpaired points
- [x] Vicenti, Figure 1, _bitmap_, paired by point shape
- [x] Valleriani21, Figure 1a, 1b, 2a, 2b (you can read the value directly from the figure, you don't have to measure coordinations)
- [x] Wang21i, Figure 2B, 2D
- [x] Wang21k, Figure 1a, 1b, 1d
- [x] Wilson21, Figure 1A, Figure 1C. _bitmap_
- [x] Wu21c, Figure 3B, Figure 3D. _bitmap_
- [x] Xie21b, Figure 1F-M
- [x] Yadav21c, Figure 1A, 1B. _bitmap_
- [x] Yadav21d, Figure 1e, 1f, 1g. _bitmap_
- [x] Yu21b, Extended Data Fig. 1 right. _bitmap_
- [x] Zani21, Figure 1A, 1B, 1C, 1D. _bitmap_
- [x] Zhang21f, Figure 3A, 3B, 3C, 4D, 4E, 4F, 4G, 5D, 5E, 5F, 5G (S8 A-F, S9 A-F, ignore for now, NTD single mutation)
- [x] Zhang21g (pending)
- [x] Zuckerman21, Figure 3, points unpaired, _bitmap_[](url)
- [x] Zuckerman21b, Figure 3 (unpaired, _vector_), or Supplementary Figure 1a,1b, 1c (paired, _bitmap_)
- [x] Ai21, Figure 3
- [x] Lusvarghi21, Figure 1A, Figure 2
- [x] Dejnirattisai21c, Figure 3
- [x] Hoffmann22, Figure 3
- [x] Meng21b, Supplementary Figure 4, Figure 1e
- [x] Carreno21b, Figure 1C to 1J, neutralization only
- [x] Bekliz21,10.1101/2021.12.28.21268491, Figure 1, Figure 2
- [x] Gruell22 (pending)
- [x] Hu21d,10.1101/2021.12.24.474110

**Comment by KaimingTao** [comment]
We plan to update aggregate data to individual data in a figure, if the figure in a paper is using vectorirzed image.

**Comment by KaimingTao** [comment]
## Other aggregate data paper (just list here)
- [] Abe21
- [] Annavajhala21b
- [] Anichini21
- [] Behrens21
- [] Betton21
- [] Caniels21
- [] Carreno21
- [] Chen21c
- [] Fernandez21
- [] Garcia-Beltran21
- [] Gonzalez21
- [] Groß21
- [] Gushchin21
- [] Hammerschmidt21
- [] Kuzmina21
- [] Leier21
- [] Li21c
- [] Lien21
- [] Marot21
- [] Molcochova21
- [] Sahin20
- [] Sahin21
- [] Spencer21
- [] Tada21
- [] Trombetta21
- [] Vacharathit21
- [] Wall21b
- [] Wilhelm21
- [] Wu21b
- [] Wu21c
- [] Yadav21c
- [] Zhang21c
- [] Zhang21d
- [] Zuckerman21

**Comment by KaimingTao** [comment]
## Complex image

- [ ] Pegu21, Figure 2A, 2B

**Comment by KaimingTao** [comment]
Previous processed:

- [ ] Sapkal21c, Figure 1A-E (paired, _bitmap_, overlap), Figure 1F, 1G, (unpaired, _bitmap_). use these figures to estimate paired data.

**Comment by zachandfox** [comment]
mccallum21c-pot.csv:

I was able to derive a set of links and points, but because many points land on the same values (or at least very close values) for a given `iso_name`  it is not possible to decide the track for any of the points that land on the same values as others. This will show in the dataset as duplications of numbered `rx_name_{link number}` identifiers for the given `iso_name` where they land on the same value as other points. I've included column id to represent the same point for a given `rx_name` + `iso_name` context.
